lib/spack/docs/getting_started.rst

```diff
@@ -459,6 +459,31 @@ specification. The operations available to modify the environment are ``set``, `
          prepend_path: # Similar for append|remove_path
            LD_LIBRARY_PATH: /ld/paths/added/by/setvars/sh
 
+Finally, if you intend to cross-compile for a different system architecture,
+you can specify the ``target``. For example, to set up an ``aarch64`` cross-compiler on an ``x86_64`` host:
+
+.. code-block:: yaml
+
+   compilers:
+   - compiler:
+       spec: gcc@9.4.0
+       paths:
+         cc: /usr/bin/aarch64-linux-gnu-gcc-9
+         cxx: /usr/bin/aarch64-linux-gnu-g++-9
+         f77: /usr/bin/aarch64-linux-gnu-gfortran-9
+         fc: /usr/bin/aarch64-linux-gnu-gfortran-9
+       flags: {}
+       operating_system: ubuntu20.04
+       target: aarch64
+       modules: []
+       environment: {}
+       extra_rpaths: []
+
+You might also need to adjust the concretizer setting to allow building for a
```

Comment:
Are there cases where this is not required?

At least for cross-microarchitecture compilations, this is actually the only change required, so perhaps consider moving it up.

Reply:
Hmm, it could be moved up before the compiler spec example, or moved to `lib/spack/docs/basic_usage.rst` under "Support for specific microarchitectures". @scheibelp what do you think?

Reply:
I agree the added content should 

* mention `spack config add concretizer:targets:host_compatible:false` first and then follow up
* eliminate the word "might" from "you might also need to" (since in fact this is necessary)